lib/Number/Compare.pm

```diff
@@ -62,28 +70,29 @@ Now this would be very pointless, if Number::Compare didn't understand
 magnitudes.
 
 The target value may use magnitudes of kilobytes (C<k>, C<ki>),
-megabytes (C<m>, C<mi>), or gigabytes (C<g>, C<gi>).  Those suffixed
-with an C<i> use the appropriate 2**n version in accordance with the
+megabytes (C<m>, C<mi>), gigabytes (C<g>, C<gi>), terabytes (C<t>, C<ti>),
+petabytes (C<p>, C<pi>), and exabytes (C<e>, C<ei>).
```

Comment:
I prefer the original 'or exabytes' here, as it's intended to signify the disjunctive nature of the set, you can't combine suffixes.

Reply:
Sorry about that, will fix it.